reservations/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.utils import timezone
+
+from utils import TimeStampModel
+
+class Reservation(models.Model):
+    reservation_type = models.CharField(max_length = 20)
+    people           = models.IntegerField()
```

Comment:
```suggestion
    people           = models.PositiveIntegerField()
```
음수를 수용할 필요가 없는 필드면 양수만  받도록 합시다~!

Reply:
넵 알겠습니다 😁